### DISCUSSION TITLE
Copilot setup crashes after entering code on WSL2/Neovim.

### DISCUSSION THREAD

**Discussion by sarahkittyy** [community/community#10555] [Copilot Conversations] [answered]
```
NVIM v0.7.0-dev+761-g0d7a97224
```

Steps to reproduce:
- Run neovim
- `:Copilot setup`
- Open browser, enter code, authorize

Neovim then hangs on the code prompt, and crashes.
![image](https://user-images.githubusercontent.com/31551546/151065501-459c33e6-2568-4094-a6d9-6e12ab7a0111.png)

**Comment by tpope** [comment, marked as answer]
The latest version features a `:Copilot! setup` variant that will allow you to open the browser manually.

**Reply by tpope** [reply]
Try visiting https://github.com/login/device and following the instructions there.

**Reply by HUAHUAI23** [reply]
many thanks, i forget to submit my authorization
